homeassistant/components/switchbot_cloud/switch.py

```diff
@@ -0,0 +1,82 @@
+"""Support for SwitchBot switch."""
+from typing import Any
+
+from switchbot_api import CommonCommands, Device, PowerState, Remote, SwitchBotAPI
+
+from homeassistant.components.switch import SwitchDeviceClass, SwitchEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import DiscoveryInfoType
+
+from . import SwitchbotCloudData
+from .const import DOMAIN
+from .coordinator import SwitchBotCoordinator
+from .entity import SwitchBotCloudEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
+    """Set up SwitchBot Cloud entry."""
+    data: SwitchbotCloudData = hass.data[DOMAIN][config.entry_id]
+    async_add_entities(
+        _make_instance(data.api, device, coordinator)
```

Comment:
```suggestion
        _async_make_entity(data.api, device, coordinator)
```

We have sync and async code in this code base so we want to prefix all functions that are safe to run in the event loop with `async` or `_async`